egs/iam/s5/local/prepare_lm.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
I think this file should be replaced by utils/format_lm.sh (or at least most of the things from this file should be replaced by that one)